src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List get() {
+        Bulldozer bulldozer1 = new Bulldozer();
+        Bulldozer bulldozer2 = new Bulldozer();
+        Bulldozer bulldozer3 = new Bulldozer();
+        List<Bulldozer> bulldozerList = new ArrayList<>();
+        bulldozerList.add(bulldozer1);
+        bulldozerList.add(bulldozer2);
+        bulldozerList.add(bulldozer3);
+        return bulldozerList;

```

Comment:
Я б засетила какие-то рандомные параметры нашим бульдозерам, но если их не сетить, то эту запись можно упростить
```suggestion
		return List.of(new Bulldozer(), new Bulldozer(), new Bulldozer());
```